### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by zephyrus1801** [community/community#21193] [Actions]
How to solve github bug when coauthoring commits

**Comment by Abhishek10351** [comment]
What exactly do you mean by github bug.

**Comment by radar-blip** [comment]
You can file a bug report here: https://support.github.com/contact/bug-report

**Comment by melvincwng** [comment]
@zephyrus1801 , submit a ticket and report your bug over [here](https://support.github.com/contact/bug-report)

Hope this clarifies.
